sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_simple.py

```diff
@@ -0,0 +1,15 @@
+# Testing Azure Packages has some additional complication/reading required.
```

Comment:
can delete this file too, it will not work since `azure.template` namespace has been renamed to `azure.eventhub.checkpointstoretable`